package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trycourier/courier",
-  "version": "2.8.0",
+  "version": "3.0.0",
```

Comment:
calling this out as breaking because we are changing the type definition. But doesn't have to be called out as a major version change.